src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -39,17 +44,58 @@ public static PackageDocumentationManager Instance
             }
         }
 
+        public void AddDynamoPaths(IPathManager pathManager)
```

Comment:
1. does this need to be public? Can it be internal?
2. can you add /// summary?